src/plone/restapi/serializer/utils.py

```diff
@@ -23,7 +23,10 @@ def uid_to_url(path):
     if suffix:
         href += "/" + suffix
     else:
-        target_object = uuidToObject(uid)
+        """ Pass unrestricted flag as true so the object is accessible.
+        At uuidToObject(), this leads to unrestrictedTraverse() to be invoked instead of restrictedTraverse().
+        """
```

Comment:
```suggestion
        # Pass unrestricted flag as true so the object is accessible.
        # At uuidToObject(), this leads to unrestrictedTraverse() to be invoked instead of restrictedTraverse().
```